device/ipv4.yaml

```diff
@@ -57,7 +59,23 @@ components:
           type: string
           format: ipv4
           default: 0.0.0.0
-        
+
+    Device.Ipv4GatewayMAC:
+      description: >- 
+        Object to set Manual Gateway MAC
+      type: object
+      properties:
+        auto_resolve:
```

Comment:
The property name enable would be more clear than auto_resolve which would then shift the description to the address property. The address property description should reflect that the property value will be used as the resolved gateway MAC.

Reply:
I'm not sure `enable` would make more sense as gateway mac would be available once arp is done . If we add `manual_gateway_mac` and then enable would make more sense as shown below. Let me know your thoughts

```
    ip2.address = "10.1.1.2"
    ip2.gateway = "10.1.1.1"
    ip2.manual_gateway_mac.enable = True
    ip2.manual_gateway_mac.address = "bb:bb:bb:bb:bb:bb"
```

Reply:
@ANISH-GOTTAPU I like this second proposal.